### PR TITLE
Doc for 1.11.2

### PR DESCRIPTION
**Pull request by dayou5168**
Doc for 1.11.2. modify format with VSCode Markdownlint. and update some content

### CODE REVIEW

**Comment by dayou5168** [comment]
Closes #1033

File: docs/mine/lotus/miner-addresses.md [hunk 1]
```diff
@@ -113,6 +113,12 @@ To set up a _control address_:
 
 Repeat this procedure to add additional addresses.
 
+5. You can Specify other actor to check control address:
```
Comment (johnnymatthews) [inline docs/mine/lotus/miner-addresses.md:116]:
```suggestion
5. You can specify other actor to check control address:
```

File: docs/mine/lotus/miner-configuration.md [hunk 2]
```diff
@@ -347,6 +354,19 @@ Batch 1:
...
+The available units are:
+
+```
+"ms": int64(Millisecond),  
+ "s":  int64(Second),  
+ "m":  int64(Minute),   
+ "h":  int64(Hour),
+```
+
+For example if you want to set the sector lifecycle to 180 days, you can multiply 180 days by 24 hours per day to get 4320 hours and set this value to "4320h0m0s".
```
Comment (johnnymatthews) [inline docs/mine/lotus/miner-configuration.md:368]:
```suggestion
For example, if you want to set the sector lifecycle to 180 days, you can multiply 180 days by 24 hours per day to get 4320 hours and set this value to `"4320h0m0s"`.
```

File: docs/mine/lotus/sector-pledging.md [hunk 3]
```diff
@@ -75,3 +75,56 @@ lotus-miner sectors mark-for-upgrade <sector number>
 ```
 
 The sector should become inactive within 24 hours after a new replacement sector has sealed. From that point, the pledged storage can be re-used for new sectors.
+
+## Inspect expiring sectors
+
+First you can check the sectors that are about to expire. Sectors that will expire within 60 days can be checked by default with the following command:
```
Comment (johnnymatthews) [inline docs/mine/lotus/sector-pledging.md:81]:
```suggestion
You can check which sectors are about to expire. Sectors that will expire within 60 days can be checked by default with the following command:
```

File: docs/mine/lotus/sector-pledging.md [hunk 4]
```diff
@@ -75,3 +75,56 @@ lotus-miner sectors mark-for-upgrade <sector number>
...
+
+```shell
+lotus-miner sectors check-expire
+```
+
+If you want to check for sectors that will expire within 33 days (669600 epoch in devnet) , add the `--cutoff` option, timing in epoch, here is an example in devnet:
```
Comment (johnnymatthews) [inline docs/mine/lotus/sector-pledging.md:87]:
```suggestion
If you want to check for sectors that will expire within 33 days (669600 epoch in devnet) , add the `--cutoff` option along with your desired epoch:
```

File: docs/mine/lotus/sector-pledging.md [hunk 5]
```diff
@@ -75,3 +75,56 @@ lotus-miner sectors mark-for-upgrade <sector number>
...
+```
+
+```
+ID  SealProof  InitialPledge  Activation                      Expiration                  MaxExpiration                 MaxExtendNow                  
+5   5          59.605 nFIL    1519 (1 day 9 hours ago)        691857 (in 4 weeks 2 days)  5257519 (in 34 weeks 3 days)  1587303 (in 10 weeks 2 days)  
+10  5          59.605 nFIL    3588 (1 day 7 hours ago)        697617 (in 4 weeks 2 days)  5259588 (in 34 weeks 4 days)  1587303 (in 10 weeks 2 days)  
+11  5          59.605 nFIL    4695 (1 day 6 hours ago)        697617 (in 4 weeks 2 days)  5260695 (in 34 weeks 4 days)  1587303 (in 10 weeks 2 days)  
+15  5          59.605 nFIL    6891 (1 day 4 hours ago)        700497 (in 4 weeks 2 days)  5262891 (in 34 weeks 4 days)  1587303 (in 10 weeks 2 days)  
+17  5          59.605 nFIL    7004 (1 day 3 hours ago)        700497 (in 4 weeks 2 days)  5263004 (in 34 weeks 4 days)  1587303 (in 10 weeks 2 days)
+```
```
Comment (johnnymatthews) [inline docs/mine/lotus/sector-pledging.md:100]:
````suggestion
```shell output
ID  SealProof  InitialPledge  Activation                      Expiration                  MaxExpiration                 MaxExtendNow                  
5   5          59.605 nFIL    1519 (1 day 9 hours ago)        691857 (in 4 weeks 2 days)  5257519 (in 34 weeks 3 days)  1587303 (in 10 weeks 2 days)  
10  5          59.605 nFIL    3588 (1 day 7 hours ago)        697617 (in 4 weeks 2 days)  5259588 (in 34 weeks 4 days)  1587303 (in 10 weeks 2 days)  
11  5          59.605 nFIL    4695 (1 day 6 hours ago)        697617 (in 4 weeks 2 days)  5260695 (in 34 weeks 4 days)  1587303 (in 10 weeks 2 days)  
15  5          59.605 nFIL    6891 (1 day 4 hours ago)        700497 (in 4 weeks 2 days)  5262891 (in 34 weeks 4 days)  1587303 (in 10 weeks 2 days)  
17  5          59.605 nFIL    7004 (1 day 3 hours ago)        700497 (in 4 weeks 2 days)  5263004 (in 34 weeks 4 days)  1587303 (in 10 weeks 2 days)
```
````

File: docs/mine/lotus/sector-pledging.md [hunk 6]
```diff
@@ -75,3 +75,56 @@ lotus-miner sectors mark-for-upgrade <sector number>
...
+
+```shell
+lotus-miner sectors check-expire
+```
+
+If you want to check for sectors that will expire within 33 days (669600 epoch in devnet) , add the `--cutoff` option, timing in epoch, here is an example in devnet:
+
+```shell with-output
+lotus-miner sectors check-expire --cutoff 669600
+```
```
Comment (johnnymatthews) [inline docs/mine/lotus/sector-pledging.md:91]:
````suggestion
```shell
lotus-miner sectors check-expire --cutoff 669600
```
````

File: docs/mine/lotus/sector-pledging.md [hunk 7]
```diff
@@ -75,3 +75,56 @@ lotus-miner sectors mark-for-upgrade <sector number>
...
+
+## Extend sectors
+
+You can extend the lifecycle of a sector with the command:
+
+```shell  practical and flexible tool:
+
+```shell
+lotus-miner sectors renew [command options] [arguments...]
+```
```
Comment (johnnymatthews) [inline docs/mine/lotus/sector-pledging.md:110]:
````suggestion
```shell
lotus-miner sectors renew [command options] [arguments...]
```
````

File: docs/mine/lotus/sector-pledging.md [hunk 8]
```diff
@@ -75,3 +75,56 @@ lotus-miner sectors mark-for-upgrade <sector number>
...
+lotus-miner sectors renew  --sector-file <your-sectorfile> --new-expiration 1555200
+```
+
+::: warning
+You have to select the sectors to renew. That means you have to specify the `--from` and `--to` option, or specify the sector file, if no sector is selected this command will have no effect.
+Format of sectorfile:  
+ 1  
+ 2  
+ ...  
+:::
```
Comment (johnnymatthews) [inline docs/mine/lotus/sector-pledging.md:130]:
````suggestion
::: warning
You have to select the sectors to renew. That means you have to specify the `--from` and `--to` option, or specify the sector file, if no sector is selected this command will have no effect.

Format of sector file:

```  
1  
2  
...
```  
:::
````

**Review by johnnymatthews — CHANGES_REQUESTED** [review]

**Comment by johnnymatthews** [comment]
This is a great PR! Sorry it took a few days to get around to reviewing it. I've submitted a few minor changes.

**Review by johnnymatthews — APPROVED** [review]

File: docs/mine/lotus/sector-pledging.md [hunk 9]
```diff
@@ -75,3 +75,58 @@ lotus-miner sectors mark-for-upgrade <sector number>
 ```
 
 The sector should become inactive within 24 hours after a new replacement sector has sealed. From that point, the pledged storage can be re-used for new sectors.
+
+## Inspect expiring sectors
+
+You can check which sectors are about to expire. Sectors that will expire within 60 days can be checked by default with the following command:
+
+```shell
```
Comment (johnnymatthews) [inline docs/mine/lotus/sector-pledging.md:83]:
````suggestion
```shell with-output
````

Comment (johnnymatthews) [inline docs/mine/lotus/sector-pledging.md:83]:
````suggestion
```shell with-output
````

**Pull request merged**